sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -97,11 +111,16 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static org.elasticsearch.cluster.metadata.IndexMetaData.INDEX_BLOCKS_WRITE_SETTING;
+import static org.elasticsearch.cluster.metadata.IndexMetaData.INDEX_ROUTING_REQUIRE_GROUP_PREFIX;
+import static org.elasticsearch.cluster.metadata.IndexMetaData.SETTING_NUMBER_OF_SHARDS;
 import static org.elasticsearch.common.settings.AbstractScopedSettings.ARCHIVED_SETTINGS_PREFIX;
 
 @Singleton
 public class AlterTableOperation {
 
+    private static final String SHRINK_PREFIX = ".shrinked";
```

Comment:
I'd prefer having a `.` at the end of the prefix as a (more human) separator.

Reply:
Afaik this is save as an index name starting with `.` will never be recognized as a partition or table. See https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/metadata/IndexParts.java#L156-L178